website/source/docs/builders/azure.html.md

```diff
@@ -57,7 +57,7 @@ builder.
 
 ### Optional:
 
--   `cloud_environment_name` (string) One of `Public`, `China`, or
+-   `cloud_environment_name` (string) One of `Public`, `China`, `German`, or
```

Comment:
Still feels like this should either be `China` and `Germany` or `Chinese` and `German`...